### PR TITLE
PG-1244 Added a limitation about KMS configuration update

### PR DESCRIPTION
**Pull request by nastena1606**
PG-1244

### Description
Added a limitation about KMS configuration update

### CODE REVIEW

File: documentation/docs/index.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Users can configure encryption differently for each database, encrypting specifi
 
 * Keys in the local keyfile are stored unencrypted.
 * System tables are currently not encrypted.
+* When updating the Key Management Store (KMS) configuration, ensure that both PostgreSQL and the KMS are up and running. For example, if you update the URL for the KMS and PostgreSQL is offline during these changes, it will not be able to start. [Talk to our experts](https://www.percona.com/about/contact) to outline the best update path for you.
```
Comment (dutow) [inline documentation/docs/index.md:27]: No, you currently can't update the configuration for an exisitng KMS, we do not have any functions to do that.

Comment (nastena1606) [reply documentation/docs/index.md:27]: Wait so https://perconadev.atlassian.net/browse/PG-1244 is not valid?

Comment (dutow) [reply documentation/docs/index.md:27]: It's valid, but the issue is bigger, Jan misunderstood part of the problem.

Comment (nastena1606) [reply documentation/docs/index.md]: Updated.

**Review by dutow — APPROVED** [review]

**Pull request merged**